ddpui/ddpdbt/dbt_service.py

```diff
@@ -97,7 +159,20 @@ def setup_local_dbt_workspace(org: Org, project_name: str, default_schema: str)
         logger.error(f"dbt init failed with {e.returncode}")
         return None, "Something went wrong while setting up workspace"
 
-    logger.info(f"starting to setup local dbt workspace at {project_dir}")
+    # setup packages.yml
+    dbtpackages_filename = Path(dbtrepo_dir) / "packages.yml"
+    with open(dbtpackages_filename, "w", encoding="utf-8") as dbtpackgesfile:
```

Comment:
even here we could copy a file from assets/

Reply:
Okay sure. Let me make that change.